src/indexing/EventIndex.js

```diff
@@ -17,6 +17,9 @@ limitations under the License.
 import PlatformPeg from "../PlatformPeg";
 import {MatrixClientPeg} from "../MatrixClientPeg";
 
+import * as Matrix from 'matrix-js-sdk';
```

Comment:
In general we should be preferring to import the classes we need, like the EventTimeline below, rather than the whole js-sdk.